scripts/table_updates/update_data_table.py

```diff
@@ -234,16 +307,11 @@ def update_redact_table(syn, redacted_table_info, full_data_table_info, logger):
     sample_table_id = master_table.loc[
         master_table["name"] == "Cancer Panel Test", "id_full"
     ].values[0]
-    curation_info = syn.tableQuery(
-        "SELECT record_id, curation_dt FROM %s" % curation_table_id
-    ).asDataFrame()
-    patient_info = syn.tableQuery(
-        "SELECT record_id, birth_year, hybrid_death_ind FROM %s" % patient_table_id
-    ).asDataFrame()
-    sample_info = syn.tableQuery(
-        "SELECT record_id, cpt_genie_sample_id, age_at_seq_report FROM %s"
-        % sample_table_id
-    ).asDataFrame()
+    # download tables
+    condition = f"cohort = '{cohort}'"
+    curation_info = utilities.download_synapse_table(syn, curation_table_id, "record_id, curation_dt", condition)
```

Comment:
Nit: Might be helpful to have 

```download_synapse_table(syn=syn, arg2=..., arg3...)```

So we know what the parameters are.

Reply:
Sure. I will add.

Reply:
Note, I will address this in the PR for GEN-1516 since it will be merged first.